README.md

```diff
@@ -64,7 +91,12 @@ argument. Finally, demonstrate calling `doSomething` with a function.**
 
 Your Answer:
 ```js
-// write code here
+function doSomething(thingToDo) {
+thingToDo
+}
+
+doSomething();
+
 ```
 
```

Comment:
Close. Couple things...

* You want to call `thingToDo` inside of your function. Otherwise, when `doSomething` is run it will reference `thingToDo`.
* When you call `doSomething`, you need to pass in some function as an argument. Otherwise, `doSomething` will do...nothing.